arelle/ModelDocument.py

```diff
@@ -1320,11 +1320,13 @@ def inlineXbrlDiscover(self, htmlElement):
         self.ixNStag = ixNStag = "{" + ixNS + "}" if ixNS else ""
         self.htmlBase = htmlBase
         ixdsTarget = getattr(self.modelXbrl, "ixdsTarget", None)
-        # load referenced schemas and linkbases (before validating inline HTML
-        for inlineElement in htmlElement.iterdescendants(tag=ixNStag + "references"):
-            if inlineElement.get("target") == ixdsTarget:
-                self.schemaLinkbaseRefsDiscover(inlineElement)
-                xmlValidate(self.modelXbrl, inlineElement) # validate instance elements
+        if not any(pluginMethod(self.modelXbrl) == False
```

Comment:
The negated negative check is confusing:
```suggestion
        if any(pluginMethod(self.modelXbrl)
```

Reply:
Seems like maybe you want them all to be true? I would use `all` in that case

Reply:
Changed to all